examples/multi_agent.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example: Multi Agent"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install SUMO."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup SUMO=1.10.0\n",
+    "!apt-get update --fix-missing\n",
+    "!apt-get install -y libspatialindex-dev\n",
+    "%pip install --upgrade pip wheel\n",
+    "%pip install eclipse-sumo==1.10.0\n",
+    "%env SUMO_HOME=/usr/local/lib/python3.7/dist-packages/sumo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install SMARTS."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install SMARTS\n",
+    "%cd ~\n",
+    "!rm -rf /content/SMARTS\n",
+    "!git clone https://github.com/huawei-noah/SMARTS /content/SMARTS\n",
+    "!cd /content/SMARTS && git checkout 'develop' && pip install .[camera-obs]\n",
+    "!echo -e \"import sys\\nsys.path.insert(0, '/content/SMARTS/')\" | python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Build the scenario."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Build scenarios\n",
+    "!scl scenario build-all --clean /content/SMARTS/scenarios/figure_eight"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define the agent."
```

Comment:
The notebook needs to restart here to update the dependencies in order to continue